sources/packages/api/src-sql/ProgramYear/Create-program-year.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS program_year(
+  id SERIAL PRIMARY KEY,
+  program_year VARCHAR(100) NOT NULL,
+  program_year_desc VARCHAR(250) NOT NULL,
+  form_name VARCHAR(100) NOT NULL,
+  active_indicator BOOLEAN NOT NULL DEFAULT TRUE,
```

Comment:
I would recommend following the same name convention already in place for the users table "is_active".